app/Actions/Place/GetPlaceItem/GetPlaceItemPresenter.php

```diff
@@ -49,7 +50,7 @@ public function __construct(
         $this->photoPresenter = $photoPresenter;
     }
 
-    public function present(GetPlaceItemResponse $placeResponse): array
+    public function present(GetPlaceItemResponse $placeResponse, GetPlaceRatingResponse $placeRatingResponse): array
```

Comment:
no, you can create another method `presentWithUserRating(GetPlaceItemResponse, ?float $rating)`